webapp/cypress/integration/loginActions.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+describe('Login actions', () => {
+    const username = 'username'
+    const email = 'username@gmail.com'
+    const password = 'password'
+
+    beforeEach(() => {
+        cy.apiResetServer()
+    })
+
+    const workspaceIsAvailable = () => {
+        cy.location('pathname').should('eq', '/')
+        cy.get('.Workspace').should('exist')
+        cy.get('.Sidebar').should('exist')
```

Comment:
Not asking for change but commenting for future reference.
I'm fine as is with the use of class names or element IDs since it's what we have currently. However, they are known to be brittle and typically tests more on implementation details rather than on user's perspective (on how the user access the app). In the future, we'll be using [@testing-library/cypress](https://testing-library.com/docs/cypress-testing-library/intro/) and take advantage of its APIs especially for queries - https://testing-library.com/docs/queries/about/#priority

Reply:
Agree, I was trying to use `testing-library/cypress` while adding more tests and it looks that we will need to change some components so that they have proper attributes.